docs/releases/updating_ez_platform.md

```diff
@@ -242,6 +242,35 @@ These steps are only relevant for some releases:
     ALTER TABLE ezuser ALTER COLUMN password_hash TYPE VARCHAR(255);
     ​```
 
+??? note "v2.2: Change from UTF8 to UTF8MB4"
+
+    Since v2.2 the character set for MySQL/MariaDB database tables changes from `utf8` to `utf8mb4` to support 4-byte characters.
+
+    To apply this change, use the following database update script:
+
+    ``` bash
+    mysql -u <username> -p <password> <database_name> < vendor/ezsystems/ezpublish-kernel/data/update/mysql/dbupdate-7.1.0-to-7.2.0.sql
+    ```
+
+    Be aware that these upgrade statements may fail due to index collisions.
+    This is because the indexes have been shortened, so duplicates may occur.
+    If that happens, you must remove the duplicates manually, and then repeat the statements that failed.
+
+    After successfully running those statements, change the character set and collation for each table.
```

Comment:
Sorry I was unclear. This needs a little more info. But I think we can get away with something like _...as described in `doc/upgrade/7.2.md`_, do you agree? Ref. https://github.com/ezsystems/ezpublish-kernel/pull/2296/files#diff-079518e72cd553a190335b8aeaaf92de

NB: That file won't contain that info until *after* https://github.com/ezsystems/ezpublish-kernel/pull/2296 has been merged. So don't merge your PR until that PR is merged. Sorry :)

Reply:
You're right, done.

Reply:
I don't see it yet 🔭

Reply:
Way to go, Dominika, forgetting to push :)

Reply:
They wrote a song about this back in the old days, before git was invented even: https://www.youtube.com/watch?v=Pmd3UiNfNkA